Src/Couchbase.Linq/Versioning/FeatureVersions.cs

```diff
@@ -27,5 +27,10 @@ internal static class FeatureVersions
         /// Version where support was added for array indexes.
         /// </summary>
         public static readonly ClusterVersion ArrayIndexes = new ClusterVersion(new Version(4, 5, 0));
+
+        /// <summary>
+        /// Version where support was added for SELECT RAW
```

Comment:
added support for SELECT RAW

Reply:
@MikeGoldsmith Not sure I follow what you're asking for here, doesn't look like a typo to me.

Reply:
You are right, had a total memory meltdown. Description is fine 👍